src/pages/groupComparison/vennDiagram/VennSimple.tsx

```diff
@@ -118,7 +91,7 @@ export default class VennSimple extends React.Component<IVennSimpleProps, {}> {
                 break;
         }
 
-        return computeVennJsSizes(combinations.map(combination=>{
+        const regions = combinations.map(combination=>{
```

Comment:
it would be nice to have a comment explaining difference between rectangles and regions.  it's not implicit in naming.